### ISSUE TITLE
browserify timeline failed

### ISSUE THREAD

**Issue by mast0r** [almende/vis#2280]
Hi, I've tried to get my own custom build with only a timeline and dataset. I used the code from the example and get the following error:

```
xxxxx:vis xx$ browserify custom.js -t babelify -o vis-custom.js -s vis
Error: Parsing file /Users/xx/vis/lib/timeline/Timeline.js: 'import' and 'export' may appear only with 'sourceType: module' (18:0)
    at Deps.parseDeps (/usr/local/lib/node_modules/browserify/node_modules/module-deps/index.js:454:28)
    at fromSource (/usr/local/lib/node_modules/browserify/node_modules/module-deps/index.js:391:44)
    at /usr/local/lib/node_modules/browserify/node_modules/module-deps/index.js:385:17
    at ConcatStream.<anonymous> (/usr/local/lib/node_modules/browserify/node_modules/concat-stream/index.js:36:43)
    at emitNone (events.js:91:20)
    at ConcatStream.emit (events.js:185:7)
    at finishMaybe (/usr/local/lib/node_modules/browserify/node_modules/concat-stream/node_modules/readable-stream/lib/_stream_writable.js:475:14)
    at endWritable (/usr/local/lib/node_modules/browserify/node_modules/concat-stream/node_modules/readable-stream/lib/_stream_writable.js:485:3)
    at ConcatStream.Writable.end (/usr/local/lib/node_modules/browserify/node_modules/concat-stream/node_modules/readable-stream/lib/_stream_writable.js:455:41)
    at DuplexWrapper.onend (/usr/local/lib/node_modules/browserify/node_modules/readable-stream/lib/_stream_readable.js:500:10)
```

The custom.js is just from the official example too:

```
exports.DataSet = require('./lib/DataSet');
exports.Timeline = require('./lib/timeline/Timeline');
```

I have updated all the necessary utilities (node etc.):
```
├── babelify@7.3.0 
├── browserify@13.1.1 
└── uglify-js@2.7.4 
```

What I am doing wrong?

**Comment by yotamberk** [comment]
@mojoaxel can you help?

**Comment by mojoaxel** [comment]
Custom builds are no longer possible as described in the README. 
@mast0r Sorry, we need to update the docs.

**Comment by mojoaxel** [comment]
I can not reproduce this anymore.
This was propably fixed with #2513 and/or #2466.
